core/src/validator.rs

```diff
@@ -623,6 +629,13 @@ impl Validator {
 
         let start_time = Instant::now();
 
+        // Run the speed check as early as possible to
+        let my_poh_hashes_per_second = if config.no_poh_speed_test {
+            None
+        } else {
+            Some(measure_poh_speed(POH_SPEED_CHECK_NUM_HASHES))
+        };
```

Comment:
nit

```suggestion
        let my_poh_hashes_per_second = (!config.no_poh_speed_test)
            .then_some(measure_poh_speed(POH_SPEED_CHECK_NUM_HASHES));
```

Reply:
This exact line has been replaced by a `match`, so marking resolved: https://github.com/anza-xyz/agave/pull/4185/commits/ac73c673c1a1991d43100831304fc4aad8a3befc